src/TableHeadings.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+
+
+const TableHeadings = (props) => {
+    return (
```

Comment:
These arrow functions can be written more simply with implicit return and no braces delimiting the body (because you just have one statement (one parent JSX node creation)): 
```const TableHeadings = props => <thead>
<tr>...
</thead>
```

There's nothing wrong with the way you've done it, however.